tools/distrib/python/grpcio_tools/grpc/tools/main.cc

```diff
@@ -31,6 +31,7 @@
 #include <google/protobuf/compiler/python/python_generator.h>
 
 #include "src/compiler/python_generator.h"
+#include "src/compiler/python_generator_helpers.h"
```

Comment:
I might have been unclear, sorry. The goal is to keep the surface between the plug-in and `grpcio-tools` as small as possible. The constraint is: the only inclusion from this file of anything under `src/compiler/*` is `src/compiler/python_generator.h`.